utils/githooks/pre-commit.d/20-isort.sh

```diff
@@ -4,6 +4,13 @@
 
 set -ue
 
+# Disable isort check because it doesn't work
+# - it needs to run with -rc
+# - it conflicts with pylint checks
+# - ~100 files are flagged with import errors
```

Comment:
what did you run to see these errors?

Reply:
I see. I think your isort is a bit old. `-rc` has been deprecated.

```
/apcv/shared/conda-envs/ai-1775/lib/python3.10/site-packages/isort/main.py:1227: UserWarning: W0501: The following deprecated CLI flags were used and ignored: -rc!
```

Reply:
You're right. My isort was 4.x :(